debian/homestead.init.d

```diff
@@ -122,8 +122,11 @@ get_settings()
         if [ -n "$scscf_uri" ]
         then
           server_name=$scscf_uri
+        elif [ -n "$scscf_prefix" ]
+        then
+          server_name="sip:$scscf_prefix.$sprout_hostname:$scscf;transport=TCP"
         else
-          server_name="sip:$(python /usr/share/clearwater/bin/bracket_ipv6_address.py $sprout_hostname):$scscf;transport=TCP"
```

Comment:
Is this right? Aren't you going to end up with "scscf.10.0.0.3"?

e.g. on a non-AWS AIO node: https://github.com/Metaswitch/clearwater-infrastructure/blob/master/debian/clearwater-auto-config-generic.init.d#L72